src/ConversionContext.h

```diff
@@ -16,7 +16,16 @@ namespace drafter {
         refract::Registry registry;
 
     public:
+        bool generateSourceMap;
```

Comment:
why not directly `const WrapperOptions&`?
Will we add duplicit member fer every change in options?